windows/src/global/delphi/general/Keyman.System.UpgradeRegistryKeys.pas

```diff
@@ -94,6 +94,8 @@ interface
   SRegKey_UpgradeBackupPath_LM = '\'+SRegKey_KeymanEngine_LM+'\Upgrade Backup\';  // I2642
   SRegKey_UpgradeBackupPath_CU = '\'+SRegKey_KeymanEngine_CU+'\Upgrade Backup\';  // I2642
 
+  SRegValue_UpgradeRunKeyman = 'Upgrade - Start Keyman with Windows';
```

Comment:
Is this key for upgrading to Keyman 14.0+? Would it use something like `SRegKey_KeymanEngine140_??`

Reply:
It's just a temporary key during upgrade, which is why it's in UpgradeRegistryKeys.pas.